CMake/DESCRIPTION.txt

```diff
@@ -0,0 +1,4 @@
+FlatBuffers is an efficient cross platform serialization library for
+C++, with support for Java, C# and Go. It was created at Google
```

Comment:
Can you take the description from the current main page? We support a lot more languages nowadays :)

Reply:
I've taken the summary from the README.md stripped of Markdown. The full readme would be excessive for description. In this case less is more.